webpack.config.js

```diff
@@ -11,6 +11,18 @@ var js_source_dirs = [
     return path.join(__dirname, file_path);
 });
 
-var bundles = recurseBundlePlugins(js_source_dirs, __dirname);
+// Mappings for libraries that we bundle in the Kolibri core app.
+
+var libs =  function (kolibri_name) {
+    return {
+        'lodash': kolibri_name + '.lib._',
+        'backbone': kolibri_name + '.lib.Backbone',
+        'loglevel': kolibri_name + '.lib.loglevel',
+        'kolibri': kolibri_name
```

Comment:
This is a convenience mapping to ensure that no plugins bundle in lodash, backbone, or loglevel if they do a direct require call on them. Instead, the reference will be converted into a reference to the copy that is on the core Kolibri object.